outputs.tf

```
@@ -0,0 +1,4 @@
+output "full_name" {
+  value       = var.name != null ? github_repository.default[0].full_name : null
```

You can safely use the output, even if it doesn't exist:

```suggestion
  value       = github_repository.default[0].full_name
```

---

modified